conf/messages.cy

```diff
@@ -133,6 +133,21 @@ otherTaxes.legend = Dewis treth neu fath o gynllun
 otherTaxes.pods = Rheoli a chofrestru cynlluniau pensiwn
 
 
+#######################################################
+##  OrganisationOnly
+#######################################################
+organisationOnly.title = #NOT TRANSLATED# Set up a new account
```

Comment:
Not sure these can have comments in them. This #NOT TRANSLATED# copy is printed on the view.